components/molecule/inputTags/src/index.js

```diff
@@ -63,7 +69,10 @@ const MoleculeInputTags = ({
     const {name} = ev.target
     ev.preventDefault()
     if (value) {
-      const tags = [...tagsFromProps, value]
+      const tags = [...tagsFromProps]
+      if (allowDuplicates || (!allowDuplicates && !isDuplicate(tags, value))) {
```

Comment:
I think you can change this condition to 
```js
if (allowDuplicates || !isDuplicate(tags, value))) {
```
it should work anyway

Reply:
@tonyghiani yes, done 😄